bin/rails_pg_restore

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env ruby
 
 require 'yaml'
+require 'erb'
 require 'optparse'
+require_relative "#{Dir.pwd}/config/environment"
```

Comment:
I wasn't sure if this was the best way to get the rails environment.

Also, I just realized this should probably be
```
require_relative File.join(Dir.pwd, 'config', 'environment')
```